deploy/cephfs/kubernetes/csi-nodeplugin-rbac.yaml

```diff
@@ -3,3 +3,46 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: cephfs-csi-nodeplugin
+  namespace: default
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cephfs-csi-nodeplugin
+rules:
+  - apiGroups: [""]
+    resources: ["nodes"]
+    verbs: ["get"]
```

Comment:
This is not required for cephfs encryption right?

Reply:
Removed